restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -390,6 +391,110 @@ private Object convertToBackendBooleanValue(boolean value, boolean numericBoolea
     return ImmutablePair.of(bindVariableList, firstLevelKeys);
   }
 
+  private String convertToJsonPtr(String path) {
+    return "/" + path.replaceAll(PERIOD_PATTERN.pattern(), "/").replaceAll("\\[(\\d+)\\]", "$1");
+  }
+
+  public List<String> writeManyDocs(
+      String authToken,
+      String keyspace,
+      String collection,
+      InputStream payload,
+      Optional<String> idPath,
+      Db dbFactory,
+      Map<String, String> headers)
+      throws IOException, UnauthorizedException {
+
+    DocumentDB db = dbFactory.getDocDataStoreForToken(authToken, headers);
+    JsonSurfer surfer = JsonSurferGson.INSTANCE;
+
+    boolean created = db.maybeCreateTable(keyspace, collection);
```

Comment:
can we extract this to some private method, isn't there some other places that uses the same as well? what about inserting the single document?